src/box/box.cc

```diff
@@ -3106,6 +3106,7 @@ box_demote(void)
 		if (raft->state != RAFT_STATE_LEADER)
 			return 0;
 	}
+	raft_update_last_demoted_term(raft);
```

Comment:
Nit: please, update the commit message, we don't stop the timer anymore.

Reply:
Fixed.